web_widget_image_webcam/static/src/js/webcam.js

```diff
@@ -139,7 +139,11 @@ var Webcam = {
 			})
 			.then( function(stream) {
 				// got access, attach stream to video
-				video.src = window.URL.createObjectURL( stream ) || stream;
+				try {
+				    video.srcObject = stream;
```

Comment:
Does this line throw an error in older browsers?

Reply:
Apparently it does. Honestly I couldn't reproduce a case when there's an error so I just followed the recommended fallback of the MDN doc, just in case (it's here if you want to check https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/srcObject)

Reply:
:+1: 
As this code deprecated, So this change is necessary. As this change is also done by the maintainer of this webcam library in the latest version.
https://github.com/OCA/web/pull/964/files#diff-c740e52ad4ba0136715ea03661ca21f6R319

Thereby, the library version is also updated for v11.0 module.
https://github.com/OCA/web/pull/964#issuecomment-473201816